android/src/main/java/com/bluechilli/flutteruploader/MethodCallHandlerImpl.java

```diff
@@ -92,6 +92,7 @@ private void enqueue(MethodCall call, MethodChannel.Result result) {
     Map<String, String> parameters = call.argument("data");
     Map<String, String> headers = call.argument("headers");
     String tag = call.argument("tag");
+    Boolean allowCellular = call.<Boolean>argument("allowCellular");
```

Comment:
is the `<Boolean>` type specifier required? Thought this could be inferred

Reply:
you are probably right